src/main/java/ast/psi/NodeConfig.java

```diff
@@ -0,0 +1,15 @@
+package ast.psi;
```

Comment:
Originally the NodeDelegateBase constructor took a few arguments to set dependent objects, however, each time I would need to add a parameter to the constructor, I would have to change the constructor of every subclass of NodeDelegateBase, which is 20+ objects. So, instead I just pass this config object in the constructor that contains the dependent objects.